Marlin/ultralcd.cpp

```diff
@@ -2486,31 +2486,28 @@ void kill_screen(const char* lcd_msg) {
       lcd_goto_screen(_lcd_calibrate_homing);
     }
 
-    // Move directly to the tower position with uninterpolated moves
-    // If we used interpolated moves it would cause this to become re-entrant
-    void _goto_tower_pos(const float &a) {
+    void _man_probe_pt(const float &lx, const float &ly) {
```

Comment:
This name implies (1) that it probes a point like `probe_pt` does, and that it has some relationship to `probe_pt`. If it doesn't probe but only moves, then use `move` or `goto` in the name instead of `probe`.